onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1316,3 +1316,292 @@ def _where_input_wrangler(
 assert NONDETERMINISTIC_OPS.issubset(
     TESTED_OPS
 ), f"{NONDETERMINISTIC_OPS - TESTED_OPS} not in TESTED_OPS"
+
+# List for different input dtype testing flag
+OPINFO_FUNCTION_TARGET_DTYPE: dict[
+    str,
+    tuple[Any, ...],
+] = {
```

Comment:
What is our plan to ensure type annotations are correct with this new table, since what is tested is now independent of the annotations ?

Reply:
sorry, not understand your question. @fatcat-z any idea?